### PR TITLE
Fix UI tweaks

### PR DESCRIPTION
**Pull request by wadeking98**
Resolves: #35
Resolves: #34 
Resolves: #33 
Resolves: #30
Resolves: #29
Resolves: #28
Resolves: #27
Resolves: #26
Resolves: #25
Resolves: #24
Resolves: #23 
Resolves: #22
Resolves: #21
Resolves: #20
Resolves: #19

### CODE REVIEW

File: client/src/pages/landing/components/Footer.tsx [hunk 1]
```diff
@@ -8,8 +8,13 @@ export const Footer: React.FC = () => {
   // const darkMode = useDarkMode()
 
   return (
-    <div className="flex dark:text-white justify-center content-center select-none my-8 pb-4 sm:my-4">
-      <p className="self-center mr-2 text-sm">Copyright &#169; 2022 Government of British Columbia</p>
+    <div>
+      <div className="flex dark:text-white justify-center content-center mt-2 select-none">
+        <a href="mailto:Ditrust@gov.bc.ca">Ditrust@gov.bc.ca</a>
```
Comment (amanji) [inline client/src/pages/landing/components/Footer.tsx:13]: should the `D` be capitalized for emails?

File: server/src/content/student/useCases/OnlineStore.ts [hunk 2]
```diff
@@ -86,7 +86,7 @@ export const OnlineStore: UseCase = {
           id: uuid(),
           type: StepType.PROOF,
           title: 'Confirm the information to send',
-          description: `BC Wallet will now ask you to confirm what to send. Notice how it will only share that you're a current student (i.e. yes or no). You don't have to share anything else for it to be trustable.`,
+          description: `BC Wallet will now ask you to confirm what to send. Notice how it will only share if the credential has expired or not—not even the expiry date itself gets shared. You don't have to share anything else for it to be trustable.`,
```
Comment (amanji) [inline server/src/content/student/useCases/OnlineStore.ts:89]: The hyphenated `not-not` seems a little odd. Should we use a comma instead?

File: client/src/components/QRCode.tsx [hunk 3]
```diff
@@ -29,7 +26,7 @@ export const QRCode: React.FC<Props> = ({ invitationUrl, connectionState, overla
 
   const renderQRCode = invitationUrl && (
     <div className={`relative ${overlay ? 'bg-none' : 'rounded-lg bg-bcgov-lightgrey p-4 m-auto'}`}>
-      <QR value={invitationUrl} size={isMobile ? 192 : isLarge ? 212 : 256} />
+      <QR value={invitationUrl} size={192} />
```
Comment (amanji) [inline client/src/components/QRCode.tsx:29]: Did you want to confirm on mobile that this renders ok?

Comment (wadeking98) [reply client/src/components/QRCode.tsx:29]: I confirmed that it renders as expected

File: client/src/pages/landing/components/Footer.tsx (hunk 1 above)
Comment (wadeking98) [reply client/src/pages/landing/components/Footer.tsx:13]: @alexgmetcalf need your input here, it is capitalized in this issue: https://github.com/bcgov/BC-Wallet-Demo/issues/21

Comment (alexgmetcalf) [reply client/src/pages/landing/components/Footer.tsx:13]: Either way, but yes, let's make it lowercase.

**Review by amanji — APPROVED** [review]

**Pull request merged**